src/components/TodoList.tsx

```diff
@@ -0,0 +1,38 @@
+import { Filter } from '../types/Filter';
+import { Todo } from '../types/Todo';
+import { TodoItem } from './TodoItem';
+
+interface TodoListProps {
+  todos: Todo[];
+  filter: Filter;
+  loading: boolean;
+}
+
+export const TodoList: React.FC<TodoListProps> = ({
+  todos,
+  filter,
+  loading,
+}) => {
+  const fileredTodos = todos.filter(todo => {
+    if (filter === Filter.active) {
+      return !todo.completed;
+    }
+
+    if (filter === Filter.completed) {
+      return todo.completed;
+    }
+
+    return true;
+  });
```

Comment:
move filtering to higher level. TodoList should receive list of todos and render it shouldn't filter them